crimsobot/utils/tarot.py

```diff
@@ -10,6 +10,41 @@
 from crimsobot.utils.tools import clib_path_join
 
 
+def list_cards(suit: Optional[str]) -> List[str]:
+    cards_in_suit = []
+
+    if suit == 'Major arcana':
```

Comment:
i'm fine with this for right now but we should really have a better way of identifying what suit something is in other than name or position in the deck.